projects/index.html

```diff
@@ -109,7 +108,122 @@ <h2 class="to-animate hero-animate-2" style="margin-bottom:10px; color:black"><s
+                    <div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box">
+					    <figure><a href="https://jour2106-group5.github.io/Air-quality/", target="_blank"><img class="img-responsive" src="img/project-thumbnail-group5.jpg" alt="project5" style="size:130%"></a></figure>
+                    </div>
+				 </div>
+				  <div class="row" id="fh5co-works">
+					<div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box"> 
+						<figure><a href="https://glotravel.github.io", target="_blank"><img class="img-responsive" src="img/project-thumbnail-group6.png" alt="project6"></a></figure>
+					</div>
+                    <div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box">
+					<br>
+					<br>
+					<h3 class="heading"><a href="https://glotravel.github.io", target="_blank">Travelling in North Korea (GloTravel)</a></h3>
+                        <p class="fh5co-sub">
+                            <br>"North Korea stay mysterious to the world. It is hard for tourist entering without joining escorted tour. Some people may think that travelling to North Korea is an indirect way to support the DPRK regime, thus refuse to go there. The website is going to explain the misunderstanding of travelling to North Korea, and give you a brief idea of visiting DPRK."
+                            <br>Members : Chungyin CHUNG Yin, Amanda LEUNG Wing Tuen, Carmen CHAN Wing Sum
+                            <br>Mentor : Rubio Chan</p>
+					</div>
+                 </div>
+				 <div class="row" id="fh5co-works">
+                    <div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box"> 
+			            <br>
+						<br>
+						<br>
+						<h3 class="heading"><a href="http://jour2106-group7.github.io/", target="_blank">Hong Kong Long-term Competitiveness Research</a></h3>
+                        <p class="fh5co-sub">
+                            <br>Members : King Hei WONG King Hei, Ian MAK Sin Hang, Kim Wong WONG Kam Ki
+                            <br>Mentor : Prof. Ho Lok Sang and Germaine Lau</p>
+					</div>
+                    <div class="col-md-6 col-sm-8 col-xs-8 col-xxs-12 text-center fh5co-work-item work-box">
+					    <figure><a href="http://jour2106-group7.github.io/", target="_blank"><img class="img-responsive" src="img/project-thumbnail-group7.jpg" alt="project7"></a></figure>
+			        </div>
+				 </div>
+				 <div class="row" id="fh5co-works">
+					 <br>
+					 <br>
+					<div class="col-md-12 col-sm-12 col-xs-12 col-xxs-12 text-center fh5co-work-item work-box">
+						<p><a href="https://juliannawqy.github.io/hkbu-jour2106.github.io/" class="btn btn-outline btn-md" target="_blank">Go to course info</a></p>
+					</div>
+				  </div>
+				</div>
+
+				<!-- The following contents are for xs and sm screen -->
+				<div class="visible-xs visible-sm">
+                     <div class="row" id="fh5co-works">
```

Comment:
This plugin helps: https://marketplace.visualstudio.com/items?itemName=lonefy.vscode-JS-CSS-HTML-formatter

Once you save the code, it is formatted automatically.